pnet_base/Cargo.toml

```diff
@@ -20,3 +20,7 @@ serde_test = "1.0.126"
 [package.metadata.docs.rs]
 # Enable the serde feature when generating docs on docs.rs, so the traits are visible
 features = ["serde"]
+
+
+[profile.release]
+lto = true
```

Comment:
Is this best practise?

Should LTO not be left to binaries, rather than being done at library level?

Reply:
fixed